### PR TITLE
Add support of the XGROUP SETID command

### LINKED ISSUE #1735: Add support of the XGROUP SETID command
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Motivation

This issue wants to implement [XGROUP SETID](https://redis.io/commands/xgroup-setid/) command like redis.

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by Yangsx-1**
This close #1735

### CODE REVIEW

File: src/commands/cmd_stream.cc [hunk 1]
```diff
@@ -258,6 +258,30 @@ class CommandXGroup : public Commander {
       return Status::OK();
     }
 
+    if (subcommand_ == "setid") {
+      if (args.size() != 5 && args.size() != 7) {
+        return {Status::RedisParseErr, errWrongNumOfArguments};
+      }
+
+      xgroup_create_options_.last_id = GET_OR_RET(parser.TakeStr());
```
Comment (torwig) [inline src/commands/cmd_stream.cc:266]: Should we check here if the received ID is in a proper format? I mean int_value-int_value/int_value/etc.

Comment (Yangsx-1) [reply src/commands/cmd_stream.cc:266]: It is checked at this [line](https://github.com/apache/kvrocks/pull/1833/files#diff-127fef0065aa75a681fdfe32519927b9c2a5724ce2de8bc5e19cdded49ac6b2eR418)

Comment (torwig) [reply src/commands/cmd_stream.cc:266]: I see, thank you.

**Review by torwig — APPROVED** [review]
LGTM

**Pull request merged**